src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -35,7 +35,7 @@
     public Subsystems(Hardware h) {
         hardware = h;
         if (CLIMB_ENABLED)
-            climbSubsystem = new ClimbSubsystem(hardware.climbFixed1, hardware.climbFixed2, hardware.climbAngled1, hardware.climbAngled2, hardware.climbAngle);
+            climbSubsystem = new ClimbSubsystem(hardware.climbFixed1, hardware.climbFixed2, hardware.climbAngle);
```

Comment:
This implies on the fixed arm has motors.  Consider correcting the name since one of the fixed motors is actually for angle.  The will reduce confusion and help with readability.